prow/config/jobs/operator.yaml

```diff
@@ -19,7 +19,6 @@ jobs:
     command: [make, mandiff]
 
   - name: manifest-apply
-    image: gcr.io/istio-testing/istio-builder:v20190829-4edc0e70
     command: [entrypoint, prow/e2e_install.sh]
```

Comment:
i find it strange there is an e2e_install call yet no e2e_install.sh in the repo...